### PR TITLE
BAH-412 Setup i18n for owa UI

### PR DESCRIPTION
**Pull request by sanishmaharjan**
[BAH-412](https://bahmni.atlassian.net/browse/BAH-412): Implement internationalization (I18N) in Bed Management UI
Please code review.
@mks-d

### CODE REVIEW

**Comment by sanishmaharjan** [comment]
@mks-d list of allowedLocales fetches from api.

File: owa/app/utilities/localeHelper.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    getNativeNameByLocaleCode = (locale) => {
+        const localeCode = this.getLocalCode(locale);
+        const localLanguages = [
+            {code: 'en', nativeName: 'English'},
+            {code: 'es', nativeName: 'Español'},
+            {code: 'fr', nativeName: 'Français'},
+            {code: 'it', nativeName: 'Italiano'},
+            {code: 'pt', nativeName: 'Português'}
+        ];
```
Comment (mks-d) [inline owa/app/utilities/localeHelper.js]: @sanishmaharjan why do you need those native names? And could you describe what this array is used for?

Comment (sanishmaharjan) [reply owa/app/utilities/localeHelper.js]: Locale API only return locale code. We need to get nativeName form somewhere. 
This is similar to https://github.com/Bahmni/default-config/blob/f38cb22cb6b2a179631a290ee89c97a885b16f26/openmrs/apps/home/locale_languages.json#L1-L5

File: owa/app/utilities/localeHelper.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+export default class LocaleHelper {
+    fetchLocales = async () => {
+        const response = await axios.get(urlHelper.apiBaseUrl() + '/session');
+        return {
+            localeCode: response.data.locale ? this.getLocalCode(response.data.locale) : 'en',
+            allowedLocales: response.data.allowedLocales ? response.data.allowedLocales : ['en']
+        };
+    };
+
+    getLocalCode = (locale) => {
```
[the remark this answers is not in the document]
Comment (sanishmaharjan) [reply owa/app/utilities/localeHelper.js]: @mks-d Done.

File: owa/app/utilities/localeHelper.js (hunk 1 above)
Comment (sanishmaharjan) [reply owa/app/utilities/localeHelper.js]: But, Locale code return from openmrs core API `/openmrs/ws/rest/v1/session`.

Comment (sanishmaharjan) [reply owa/app/utilities/localeHelper.js]: How about using https://www.npmjs.com/package/locale-code javascript library.

Comment (sanishmaharjan) [reply owa/app/utilities/localeHelper.js]: @mks-d implement iso-639-1 to get language native name.

**Comment by sanishmaharjan** [comment]
@mks-d jest failed issue resolved.

**Pull request merged**